openshift/templates/email-verification-agent/email-verification-agent-deploy.yaml

```diff
@@ -44,15 +91,41 @@ objects:
         kind: Service
         name: ${NAME}
         weight: 100
+  
+  - kind: Route
```

Comment:
What's the reason behind the `admin` API being exposed publicly?

Reply:
This was my suggestion: it makes registering/managing new agents easier as a tunnel is not required - a public PROTECTED endpoint can be hit instead. This is what we have been doing for other services as well.